screens/GuitarScreen.js

```diff
@@ -13,18 +21,35 @@ const styles = StyleSheet.create({
 export function GuitarScreen(props) {
   // console.log(props.navigation.state.params);
   const song = props.navigation.state.params.song;
+  const scale = (num, in_min, in_max, out_min, out_max) => {
+    return ((num - in_min) * (out_max - out_min)) / (in_max - in_min) + out_min;
+  };
+
+  let index = 4;
+
+  const handleScroll = event => {
+    index = Math.floor(
+      scale(event.nativeEvent.contentOffset.y, 0, 2290, 0, song[0].line.length)
+    );
```

Comment:
Index needs to be a state variable and it should work.

`[index, setIndex] = useState(0)`

and then in this handler:
`setIndex(Math.floor(...))`

Reply:
Got it! thanks 👍